Hashlink.js

```diff
@@ -23,45 +23,44 @@ export class Hashlink {
    * fetched, transformed, and encoded into a hashlink. If a data parameter
    * is provided, the hashlink is created from the data.
    *
-   * @param {Object} options - The options for the create operation.
-   * @param {Uint8Array} [options.data] - The data associated with the given
+   * @param {Uint8Array} [data] - The data associated with the given
    *   URL. If provided, this data is used to create the cryptographic hash.
-   * @param {Array} options.codecs - One or more codecs that should be used
+   * @param {Array<object>} codecs - One or more codecs that should be used
    *   to encode the data.
-   * @param {Array} [options.urls] - One or more URLs that contain the data
+   * @param {string|Array} [url] - One or more URLs that contain the data
```

Comment:
Should it be `{string|Array<string>}`?

Reply:
It should, yeah. Thanks, I'll make the edit.